code/modules/assembly/doorcontrol.dm

```diff
@@ -47,15 +49,14 @@
 	for(var/obj/machinery/door/airlock/D in airlocks)
 		if(D.id_tag == src.id)
 			if(specialfunctions & OPEN)
-				spawn(0)
-					if(D)
-						if(D.density)	D.open()
-						else			D.close()
-					return
```

Comment:
This caused a bug fixed by https://github.com/tgstation/tgstation/pull/21125

@neersighted please don't remove spawns for no reason.